consensus/spos/bls/subroundEndRound.go

```diff
@@ -396,7 +396,7 @@ func (sr *subroundEndRound) broadcastMiniBlocksAndTransactions() error {
 		return nil
 	}
 
-	go sr.broadcast(miniBlocks, transactions, core.ExtraDelayForBroadcastBlockInfo)
+	go sr.broadcast(miniBlocks, transactions, 0)
```

Comment:
This reintroduces the problem of receiving miniblocks before header. A slight delay is still needed here

Reply:
It is solved as peer mini blocks from meta were not rejected anymore

Reply:
If we want to white list them we should apply what I said in point 3. drom description

Reply:
All miniblocks should be whitelisted